src/UserSpaceInstrumentation/AccessTraceesMemory.cpp

```diff
@@ -43,7 +43,7 @@ using orbit_base::ReadFileToString;
   do {
     // Pack 8 byte for writing into `data`.
     uint64_t data = 0;
-    std::memcpy(&data, bytes.data() + pos , sizeof(uint64_t));
+    std::memcpy(&data, bytes.data() + pos, sizeof(uint64_t));
```

Comment:
absl::bit_cast<> does the same if I am not mistaken. It does require the data to be aligned though, I guess it should be in this case?

Reply:
absl::bit_cast only works with types of identical size:

https://github.com/abseil/abseil-cpp/blob/a76698790753d2ec71f655cdc84d61bcb27780d4/absl/base/casts.h#L42

So I'll stick with the memcpy.